freqtrade/tests/test_persistence.py

```diff
@@ -426,6 +426,7 @@ def test_migrate_new(mocker, default_conf, fee, caplog):
                                 max_rate FLOAT,
                                 sell_reason VARCHAR,
                                 strategy VARCHAR,
+                                ticker_interval INTEGER,
```

Comment:
almost ... please also add a check below to verify that the new column has the content assumed (null) - and isn't given some "random" value during the transition ... ;) 
Rest of the Persistence stuff looks good now :+1: